tools/text_processing_deployment/export_grammars.sh

```diff
@@ -37,6 +37,8 @@ MODE="export"
 OVERWRITE_CACHE="True" # Set to False to re-use .far files
 FORCE_REBUILD="False" # Set to True to re-build docker file
 WHITELIST=None # Path to a whitelist file, if None the default will be used
+FAR_PATH=$(pwd) # Path where the grammars should be written
+SKIP_FAR_CREATION="False"
```

Comment:
What's the difference between `SKIP_FAR_CREATION` and `OVERWRITE_CACHE`?

Reply:
SKIP_FAR_CREATION (skips creating `tokenize_and_classify.far` and `verbalize.far`). Overwrite Cache overwrites cache files (ones used in pynini). Basically if `OVERWRITE_CACHE=True`, `SKIP_FAR_CREATION=False`. It's for a very specify usecase where cache files exist and we still want to overwrite the two far files required for sparrowhawk.

Reply:
`It's for a very specify usecase where cache files exist and we still want to overwrite the two far files required for sparrowhawk.` - this is done by OVERWRITE_CACHE, no?

Reply:
Not exactly. The tokenizer and verbalizer far files are created from the cache. If pynini tests ran but export failed due to some other reason, we may not want to overwrite the cache file.